### DISCUSSION TITLE
Altering web panel host location from pastebin

### DISCUSSION THREAD

**Discussion by BushDid911lol** [UnamSanctam/UnamWebPanel#137] [Q&A]
heyyyy qucik question, 

Let's say the server im using to host the web panel gets shut down :( 

Can I change the website url in connected pastebin doc to a new server (the endpoint.php)?

if so ,, what is the line to add? 
thanks brosef.

**Comment by UnamSanctam** [comment]
Yes you can add `"api-endpoint": "API Endpoint URL"`

**Reply by Storaqe** [reply]
> Yes you can add `"api-endpoint": "API Endpoint URL"`

in the remote configuration? cause my panel from 000webhost was put to sleep for idk what, not banned.

also what free host would u recommend for it?

**Reply by UnamSanctam** [reply]
Yes in the "Remote Configuration", free 000webhost sites are put to sleep after they have too much traffic (since they are free). There isn't really free host that I know of without limitations since it costs money/resources for them to host it.
